bonus_file_with_defs.rb

```diff
@@ -0,0 +1,274 @@
+#################################################################################
+###########################    General methods    ###############################
+#################################################################################
+require 'fileutils'
+
+def puts_something
+  puts ""
+  puts "---List of words that you will meet in tests---"
+end
+
+def read_this_file (file_to_read)
+  @array_with_strings_from_txt = []
+  File.open(file_to_read) do |review_file|
+    @array_with_strings_from_txt = review_file.readlines
+    puts @array_with_strings_from_txt
+  end
+end
+
+def start_test_1_or_not
```

Comment:
This name of method isn't correct due to rules of Clean Code. You can't use "or" or "and" in the names of your method.